Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -7,14 +7,13 @@ final class DemoMerchantAPI {
 
     private init() {}
 
+
     /// This function replicates a way a merchant may go about creating an order on their server and is not part of the SDK flow.
-    /// - Parameters:
-    ///   - orderParams: the parameters to create the order with
-    ///   - completion: a result object vending either the order or an error
-    func createOrder(orderParams: CreateOrderParams, completion: @escaping ((Result<Order, URLResponseError>) -> Void)) {
+    /// - Parameter orderParams: the parameters to create the order with
+    /// - Returns: a result object vending either the order or an error
```

Comment:
Technically, we aren't returning a result object anymore, so can probably update this docs string